test/basekernels/periodic.jl

```diff
@@ -8,6 +8,7 @@
     @test k(v1, v2) ≈ exp(-0.5 * sum(abs2, sinpi.(v1 - v2) ./ r))
     @test k(v1, v2) == k(v2, v1)
     @test PeriodicKernel(3)(v1, v2) == PeriodicKernel(; r=ones(3))(v1, v2)
+    @test isa(PeriodicKernel(r=Float32.(r))(Float32.(v1), Float32.(v2)), Float32)
```

Comment:
Based on the logs JuliaFormatter would like
```suggestion
    @test isa(PeriodicKernel(; r=Float32.(r))(Float32.(v1), Float32.(v2)), Float32)
```
Alternatively, possibly a bit easier to read:
```suggestion
    @test PeriodicKernel(; r=Float32.(r))(Float32.(v1), Float32.(v2)) isa Float32
```